src/cryptoadvance/specter/util/bitcoind_setup_tasks.py

```diff
@@ -187,7 +169,7 @@ def setup_bitcoind_directory_thread(specter=None, quicksync=True, pruned=True):
             logger.info("No success connecting to Bitcoin Core")
         specter.check()
         specter.reset_setup("bitcoind")
-        app.specter.setup_status["stage"] = 5
+        specter.setup_status["waiting_bitcoind_datadir_setup"] = False
```

Comment:
These setup-states are a bit confusing. Can you document the data-structure a bit better, at best in the method-doc of the update_setup_status method?
https://github.com/cryptoadvance/specter-desktop/blob/13d3224f9b53dbf3ec1fd6d77e720d34775b26ef/src/cryptoadvance/specter/specter.py#L586-L594

BTW, is this still correct:
https://github.com/cryptoadvance/specter-desktop/blob/13d3224f9b53dbf3ec1fd6d77e720d34775b26ef/src/cryptoadvance/specter/specter.py#L187-L198